bucket/azure-kubelogin.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "0.0.11",
+    "description": "A client-go credential (exec) plugin implementing azure authentication.",
```

Comment:
```suggestion
    "description": "A client-go credential (exec) plugin implementing Azure authentication.",
```

Reply:
I've changed this, but it's now not the same as the official docs on the repo README.